crates/forge/bin/cmd/getsolc.rs

```diff
@@ -0,0 +1,50 @@
+use clap::{Parser, ValueHint};
+use eyre::Result;
+use foundry_compilers::{artifacts::Source, utils, Graph, Project};
+use std::path::PathBuf;
+use yansi::Paint;
+
+/// CLI arguments for `forge get-solc`.
+#[derive(Clone, Debug, Parser)]
+pub struct GetSolcArgs {
+    /// The path of the project to get solc for.
+    #[arg(value_hint = ValueHint::DirPath, default_value = ".", value_name = "PATH")]
+    path: PathBuf,
+}
+
+impl GetSolcArgs {
+    pub fn run(self) -> Result<()> {
+        let GetSolcArgs { path } = self;
+        get_solc_from_path(path)?;
```

Comment:
this should load the `Config` using the the path.
from which we can get the ProjectPaths

Reply:
how can I get this? also I think resolve_sources adds the sources if they are not in the project.

Reply:
> this should load the `Config` using the the path. from which we can get the ProjectPaths

i did this but now if you specify a path for a folder which uses a single solc, the output is always something like this `Solcs 0.8.19, 0.8.21, 0.8.24` no matter the given path, which wasn't the case earlier.